src/nft/hooks/useBag.ts

```diff
@@ -104,6 +142,29 @@ export const useBag = create<BagState>()(
           return { itemsInBag: itemsCopy }
         })
       },
+      removeAssetsFromBag: (assets) => {
+        set(({ itemsInBag }) => {
+          if (get().isLocked) return { itemsInBag: get().itemsInBag }
+          if (itemsInBag.length === 0) return { itemsInBag: [] }
+          const itemsCopy = itemsInBag.filter((item) => !assets.some((asset) => asset.id === item.asset.id))
+          return { itemsInBag: itemsCopy }
+        })
```

Comment:
Do we need a seperate function or can we either call `removeAsset` multiple times or replace calls to `removeAsset` with `removeAsset` and an array of length 1?

Reply:
I like that - also going to do the same for addAssetToBag